### ISSUE TITLE
Error with sandbox remote build

### ISSUE THREAD

**Issue by akshayMpatel** [apptainer/singularity#5459]
### Version of Singularity:
3.5.3-1.1.el7

### How did you install Singularity
rpm

### What OS/distro are you running

NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
...

### With command:
`singularity build --remote --sandbox container container.def`

### Expected behavior
create the sandbox directory

### Actual behavior
...
INFO:    Creating SIF file...
INFO:    Build complete: /tmp/image-780251018
WARNING: Skipping container verifying
 1.93 GiB / 1.93 GiB  100.00% 46.06 MiB/s 42s01s20s
INFO:    Starting build...
FATAL:   While performing build: conveyor failed to get: failed to retrieve path for /tmp/remote-build-914678969: lstat /tmp/remote-build-914678969: no such file or directory

**Comment by dtrudg** [comment]
This is not currently handled well by the CLI.

You can get a sandbox in two steps:

```
singularity build --remote container.sif container.def
singularity build --sandbox container_sandbox container.sif
```

**Comment by dtrudg** [comment]
Resolved in 3.6.3
